src/util/opaque_pointer.rs

```diff
@@ -13,13 +13,23 @@ pub struct OpaquePointer(*mut c_void);
 unsafe impl Sync for OpaquePointer {}
 unsafe impl Send for OpaquePointer {}
 
+impl Default for OpaquePointer {
+    fn default() -> Self {
+        Self::UNINITIALIZED
+    }
+}
+
 impl OpaquePointer {
     pub const UNINITIALIZED: Self = Self(0 as *mut c_void);
 
     pub fn from_address(addr: Address) -> Self {
         OpaquePointer(addr.to_mut_ptr::<c_void>())
     }
 
+    pub fn to_address(self) -> Address {
```

Comment:
It is used for jikesrvm binding to get the internal data from a tls pointer. I removed this method so that mmtk-core will not be read/write to `OpaquePointer`. In jikesrvm binding, it transmutes the `OpaquePointer` to `Address` to read tls data.